pkg/apis/camel/v1/build_types.go

```diff
@@ -147,6 +147,7 @@ const (
 // Change format to 'duration' when CRD uses OpenAPI spec v3 (https://github.com/OAI/OpenAPI-Specification/issues/845)
 // +kubebuilder:printcolumn:name="Duration",type=string,JSONPath=`.status.duration`,description="The build last execution duration"
 // +kubebuilder:printcolumn:name="Attempts",type=integer,JSONPath=`.status.failure.recovery.attempt`,description="The number of execution attempts"
+// +kubebuilder:resource:path=builds,scope=Namespaced,shortName=it,categories=kamel;camel
```

Comment:
Could you append to the existing `kubebuilder:resource` marker instead of duplicating it?

Reply:
done, thanks :)